apps/pie-docs/test/playwright/fixtures.js

```diff
@@ -6,13 +6,13 @@ const AxeBuilder = require('@axe-core/playwright').default;
 // This new "test" can be used in multiple test files, and each of them will get
 // a consistently configured AxeBuilder instance.
 exports.test = base.test.extend({
-    makeAxeBuilder: async ({ page }, use) => {
+    makeAxeBuilder: [async ({ page }, use) => {
```

Comment:
What's with the array?

Reply:
Previously the `timeout` value would've applied to all test fixtures defined. This array means that the timeout only applies to the `makeAxeBuilder` function 👍🏼

Reply:
Do you have a link to the docs where it shows it'll work when passed an array like this? The ones I've managed to find don't really show it

Reply:
Here ya go! - https://playwright.dev/docs/test-fixtures#worker-scoped-fixtures

```
  account: [async ({ browser }, use, workerInfo) => {
    // Unique username.
    const username = 'user' + workerInfo.workerIndex;
    const password = 'verysecure';

    // Create the account with Playwright.
    const page = await browser.newPage();
    await page.goto('/signup');
    await page.getByLabel('User Name').fill(username);
    await page.getByLabel('Password').fill(password);
    await page.getByText('Sign up').click();
    // Make sure everything is ok.
    await expect(page.locator('#result')).toHaveText('Success');
    // Do not forget to cleanup.
    await page.close();

    // Use the account value.
    await use({ username, password });
  }, { scope: 'worker' }],
```